samples/net/sockets/echo_client/Kconfig

```diff
@@ -46,4 +46,11 @@ config NET_SAMPLE_IFACE3_VLAN_TAG
 	  Set VLAN (virtual LAN) tag (id) that is used in the sample
```

Comment:
The commit "samples: net: echo_client: Add Kconfig option to run a number of times" still says
```
For the echo_client, its prj.conf file defines
the value as 10, in which time the functionality is demonstrated
and the sample can be run in a meaningful way.
```
But as we removed the value from prj.conf this sentence can be removed.

Reply:
Done. And the license.